charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -119,20 +104,21 @@ spec:
         - name: scalardb-cluster-database-properties-volume
           configMap:
             name: {{ include "scalardb-cluster.fullname" . }}-node-properties
-        {{- if .Values.scalardbCluster.tls.caRootCertSecret }}
-        - name: scalardb-cluster-tls-ca-root-volume
+        {{- if and .Values.scalardbCluster.tls.enabled .Values.scalardbCluster.tls.certManager.enabled }}
+        - name: scalardb-cluster-tls-volume
           secret:
-            secretName: {{ .Values.scalardbCluster.tls.caRootCertSecret }}
+            secretName: {{ include "scalardb-cluster.fullname" . }}-tls-cert
```

Comment:
If you use cert-manager, it creates one secret resource that includes all private keys and certificates. So, it's enough to mount that one secret here.